pom.xml

```diff
@@ -98,7 +98,7 @@
     <jackson-bom.version>2.12.6.20220326</jackson-bom.version>
     <antlr.version>3.5.2</antlr.version>
     <!-- Only used for tests with HBase 2.1-2.4 -->
-    <reload4j.version>1.2.19</reload4j.version>
+    <reload4j.version>1.2.21</reload4j.version>
```

Comment:
This doesn't seem related to this PR

Reply:
Yes, it is not, but I was getting build error initially which got fixed with this change. I will try again without this change.

Reply:
I was able to compile without this change, so reverted it.

Reply:
I forgot that the issue was not with the compilation but the ITs. Without changing this version, I am getting the below exception:

```
java.lang.NoSuchFieldError: tlm
	at org.apache.log4j.MDCFriend.fixForJava9(MDCFriend.java:11)
	at org.slf4j.impl.Log4jMDCAdapter.<clinit>(Log4jMDCAdapter.java:38)
	at org.slf4j.impl.StaticMDCBinder.getMDCA(StaticMDCBinder.java:59)
	at org.slf4j.MDC.bwCompatibleGetMDCAdapterFromBinder(MDC.java:99)
	at org.slf4j.MDC.<clinit>(MDC.java:108)
	at org.apache.zookeeper.ClientCnxn$SendThread.startConnect(ClientCnxn.java:1080)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1139)
```

This got fixed when I bumped up this version. Could you check if you are able to run this IT in IntelliJ with the old version?